.github/workflows/ci-workflow.yml

```diff
@@ -117,8 +118,13 @@ jobs:
 
   demo-app-backstop:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        test: [1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
This is set so that backstop runs multiple times on this branch to make it easier to flush out any remaining reliability issues.